### ISSUE TITLE
ANTs Subprocess Error

### ISSUE THREAD

**Issue by shlrley** [CobraLab/documentation#9]
![Screen Shot 2021-02-11 at 3 28 32 PM](https://user-images.githubusercontent.com/70875076/107715147-daada880-6c8b-11eb-8872-cdc7c7b3277a.png)

Hi, 

I'm running the ANTs procedure on 543 subjects. It has been going well, but I ran into an error where it says that the directory "**output/jacobians/resampled/COMPLETE**" cannot be found. (Please see image attached)

It looks like the only directory inside of my **output/jacobians** directory is one called "**overall**"

Is this a directory I am supposed to create myself? Or is there something else I should fix? 

Thank you for your help!

**Comment by gdevenyi** [comment]
Please provide the command command you are running.

**Comment by shlrley** [comment]
Here are the contents inside of **my-dbm-run-script.sh**:

#!/bin/bash

module load cobralab/2019b

twolevel_ants_dbm/twolevel_dbm.py --modelbuild-command ${QUARANTINE_PATH}/twolevel-dbm/niagara2-antsMultivariateTemplateConstruction2.sh \
        --rigid-model-target mni_extracted_crop.nii.gz --cluster-type=slurm 1level input.csv 2>&1 | tee -a dbm_logfile.log

**Comment by gdevenyi** [comment]
Please post the full contents of `dbm_logfile.log`

**Comment by shlrley** [comment]
![Screen Shot 2021-02-12 at 8 22 57 AM](https://user-images.githubusercontent.com/70875076/107786860-9cf06480-6d0b-11eb-8ace-4701e01fa7b2.png)

Hi, here are the contents of dbm_logfile.log. 

I believe the first line (file mni_extracted_crop.nii.gz does not exist .) applies to a previous error that I have already fixed (I added mni_extracted_crop.nii.gz to my directory). I think it is just the second line that applies to this error.
